plugins/source/gcp/codegen/recipes/aiplatform.go

```diff
@@ -0,0 +1,76 @@
+package recipes
+
+import (
+	aiplatform "cloud.google.com/go/aiplatform/apiv1"
+	pb "cloud.google.com/go/aiplatform/apiv1/aiplatformpb"
+	locationpb "google.golang.org/genproto/googleapis/cloud/location"
+)
+
+func init() {
+	resources := []*Resource{
```

Comment:
This is the bulk of the PR, the other are changes to support listing locations before listing the actual resources